airflow/providers/postgres/hooks/postgres.py

```diff
@@ -53,13 +58,13 @@ class PostgresHook(DbApiHook):
     default_conn_name = 'postgres_default'
     supports_autocommit = True
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
         self.schema = kwargs.pop("schema", None)
         self.connection = kwargs.pop("connection", None)
         self.conn = None
```

Comment:
```suggestion
        self.conn: Optional[Any] = None
```
Should solve the issue from CI. I'm not exactly sure what `self.conn` will became but it should be some object according to this line:
https://github.com/apache/airflow/blob/12a652f5344c7f03c3d780556ca1829b235fdb2d/airflow/providers/postgres/hooks/postgres.py#L104